src/set_theory/game/impartial.lean

```diff
@@ -78,14 +78,14 @@ begin
   introsI hG hH,
   rw impartial_def,
   refine ⟨equiv_trans (add_congr (neg_equiv_self _) (neg_equiv_self _))
-    (neg_add_relabelling _ _).equiv.symm, λ i, _, λ i, _⟩,
-  { rcases left_moves_add_cases i with ⟨j, rfl⟩ | ⟨j, rfl⟩,
+    (neg_add_relabelling _ _).equiv.symm, λ k, _, λ k, _⟩,
+  { apply left_moves_add_cases k,
```

Comment:
Does `induction k using pgame.left_moves_add_cases` work?

Reply:
I get an error
```
invalid user defined recursor, type of the major premise 'k' does not contain the recursor parameter 'x'
```

Reply:
What if you make `k` the last argument rather than the second?

Reply:
Same error still.